### DISCUSSION TITLE
Testing Angular in CodeSandbox

### DISCUSSION THREAD

**Discussion by Chhekur** [codesandbox/codesandbox-client#4531] [General]
I was trying to write spec for Angular in CodeSandbox-client
```ts
  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [FootballMatchesComponent]
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(FootballMatchesComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });
```

but I am getting fixture is undefined.

So is there any specific way to write test for angular in csb-client or is it possible to test angular in csb-client ?

**Comment by CompuIves** [comment]
Hey @Chhekur! Do you have an example sandbox? Then I can check why fixture itself is undefined.

**Reply by Chhekur** [reply]
no worries, the issue is resolved ; )
